.buildkite/pipeline.yml

```diff
@@ -171,7 +171,7 @@ steps:
     artifact_paths:
       - "build/TEST-go-unit.cov"
     agents:
-      image: "golang:1.20.10"
+      image: "golang:1.22.6"
```

Comment:
Related, I think this is being omitted from the action that auto-bumps the go version https://github.com/elastic/elastic-agent/blob/main/.ci/updatecli/updatecli-bump-golang.yml

Reply:
IMHO, we should not have a pinned version in there like that or if we endup doing it like that, to have it be bumped with the automation.

Reply:
Tracking with https://github.com/elastic/ingest-dev/issues/3969